src/libical-glib/api/i-cal-derived-property.xml

```diff
@@ -133,1717 +133,1717 @@
-        <comment xml:space="preserve">Create a new #ICalProperty.</comment>
+        <comment xml:space="preserve">Creates a new #ICalProperty.</comment>
     </method>
     <method name="i_cal_property_set_tzname" corresponds="icalproperty_set_tzname" kind="set" since="1.0">
         <parameter type="ICalProperty *" name="prop" comment="The #ICalProperty to be set"/>
-        <parameter type="const gchar *" name="v" comment="The tzname."/>
-        <comment>Set the tzname for the #ICalProperty.</comment>
+        <parameter type="const gchar *" name="v" comment="The tzname"/>
+        <comment>Sets the tzname for the #ICalProperty.</comment>
     </method>
     <method name="i_cal_property_get_tzname" corresponds="icalproperty_get_tzname" kind="get" since="1.0">
-        <parameter type="ICalProperty *" name="prop" comment="The #ICalProperty to be queried."/>
+        <parameter type="ICalProperty *" name="prop" comment="The #ICalProperty to be queried"/>
         <returns type="const gchar *" annotation="transfer none" comment="Get the tzname of #ICalProperty."/>
-        <comment>Get the tzname of #ICalProperty.</comment>
+        <comment>Gets the tzname of #ICalProperty.</comment>
     </method>
     <method name="i_cal_property_new_tzoffsetfrom" corresponds="icalproperty_new_tzoffsetfrom" kind="constructor" since="1.0">
-        <parameter type="gint" name="v" comment="The tzoffsetfrom."/>
+        <parameter type="gint" name="v" comment="The tzoffsetfrom"/>
         <returns type="ICalProperty *" annotation="transfer full" comment="The newly create #ICalProperty"/>
-        <comment xml:space="preserve">Create a new #ICalProperty.</comment>
+        <comment xml:space="preserve">Creates a new #ICalProperty.</comment>
     </method>
     <method name="i_cal_property_set_tzoffsetfrom" corresponds="icalproperty_set_tzoffsetfrom" kind="set" since="1.0">
         <parameter type="ICalProperty *" name="prop" comment="The #ICalProperty to be set"/>
-        <parameter type="gint" name="v" comment="The tzoffsetfrom."/>
-        <comment>Set the tzoffsetfrom for the #ICalProperty.</comment>
+        <parameter type="gint" name="v" comment="The tzoffsetfrom"/>
+        <comment>Sets the tzoffsetfrom for the #ICalProperty.</comment>
     </method>
     <method name="i_cal_property_get_tzoffsetfrom" corresponds="icalproperty_get_tzoffsetfrom" kind="get" since="1.0">
-        <parameter type="ICalProperty *" name="prop" comment="The #ICalProperty to be queried."/>
+        <parameter type="ICalProperty *" name="prop" comment="The #ICalProperty to be queried"/>
         <returns type="gint" comment="Get the tzoffsetfrom of #ICalProperty."/>
-        <comment>Get the tzoffsetfrom of #ICalProperty.</comment>
+        <comment>Gets the tzoffsetfrom of #ICalProperty.</comment>
     </method>
     <method name="i_cal_property_new_tzoffsetto" corresponds="icalproperty_new_tzoffsetto" kind="constructor" since="1.0">
```

Comment:
tzoffsetto is really hard to read here and below in the comments. I don't know about the features here, but can it i.e. be wrapped in asterisks to improve readability?

The same accounts true for more similar values in this file.

Reply:
I don't get what you mean by this.

Reply:
I think @dothebart means that "tzoffestfrom/tzoffsetfo" being lower case and looking like English words -- but they aren't; they are properties of the VTIMEZONE component.  This isn't doxygen and I don't know how to indicate that "tzoffsetfoo" has some special meaning.  asterisks or quotes or something might be an improvement.  but no big deal either way.

Reply:
I could wrap it in a `@a` or `@p` to make it stand out from the rest of the text. We do that in other places for parameters, so it would be consistent. (Making that more universal is maybe something I could do, but I'll leave that for later since it's not a big deal)

Reply:
right. leave that for another follow-on set of patches.  this set is already large enough.

Reply:
Yes, thanks that was what I meant.